README.md

```diff
@@ -42,10 +56,16 @@ Those functionnalities are provided through two submodules:
 
 This project requires [Java v11+](URL) {...} [Gradle v7.3.1+](URL) {...}
 
-Use [{package manager or plateform}][install-tool] to install {project's name}:
+Use Gradle Build Tool to build, install and test the project. To build project run:
+
+```bash
+gradle build
+```
+
+In order to run tests use:
 
 ```bash
-{install-command}
+gradle clean test
```

Comment:
This section should explain how to use the SDK as an end user, not how to contribute to it.